packages/cashier/src/containers/routes/error-component/error-component.tsx

```diff
@@ -2,8 +2,12 @@ import React from 'react';
 import { useHistory } from 'react-router-dom';
 import { PageErrorContainer } from '@deriv/components';
 import { routes } from '@deriv/shared';
-import { TRootStore } from '../../../types';
 import { localize, Localize } from '@deriv/translations';
+import { TCoreStores } from '@deriv/stores/types';
+
+type TErrorComponentProps = TCoreStores['common']['error'] & {
```

Comment:
I had to fix this type because I had to fix types of common-store and its error property in types file